maintain_pim/bug_qualification/index.rst

```diff
@@ -56,7 +56,7 @@ When you notice slownesses, displaying pages and/or during import/exports, you a
 
 * Use the Symfony's profiler timeline tab to identify what part of the execution takes most of the time.
 * Use `blackfire.io <https://blackfire.io/>`_ in order to profile the requests which take too much time.
-* Use MySQL built in "SHOW PROCESSLIST" query to identify queries which take too long to run (See: `MySQL documentation: SHOW PROCESSLIST Syntax <https://dev.mysql.com/doc/refman/5.6/en/show-processlist.html>`_).
+* Use MySQL built in "SHOW PROCESSLIST" query to identify queries which take too long to run (See: `MySQL documentation: SHOW PROCESSLIST Syntax <https://dev.mysql.com/doc/refman/8.0/en/show-processlist.html>`_).
```

Comment:
I changed this in the hope that it would fix the CircleCI 403 error, but I think that it is caused by the MySQL docs website rejecting crawlers (including CircleCI).

In any case, we use MySQL v8 in PaaS, so we should update this anyway.